tool/tsh/tsh.go

```diff
@@ -2384,7 +2384,14 @@ func accessRequestForSSH(ctx context.Context, tc *client.TeleportClient) (types.
 		return nil, trace.Wrap(err)
 	}
 	req.SetLoginHint(tc.HostLogin)
-	return req, nil
+
+	req.SetDryRun(true)
+	req.SetRequestReason("Dry run, this request will not be created. If you see this, there is a bug.")
+	defer req.SetDryRun(false)
+	defer req.SetRequestReason("")
+	return req, trace.Wrap(tc.WithRootClusterClient(ctx, func(clt auth.ClientI) error {
+		return trace.Wrap(clt.CreateAccessRequest(ctx, req))
+	}))
```

Comment:
Won't this potentially return a `non-nil, non-nil` pair? Is that intended?

Even so, I'd prefer if this was written a bit more normally.